src/assets/i18n/en_US.json

```diff
@@ -137,6 +138,9 @@
   "account.costcenter.manager.label": "Cost center manager",
   "account.costcenter.name.error.required": "Please enter a cost center name.",
   "account.costcenter.name.label": "Name",
+  "account.costcenter.net.label": "Net",
+  "account.costcenter.price.type.title": "Budget calculation",
+  "account.costcenter.profile.price.type.label": "Budget calculation in",
```

Comment:
Which value will follow after "in"? If it is "gross" of "account.costcenter.gross.label" and "net" of "account.costcenter.net.label", they need to be spelled in lower case.

Reply:
Yes, the gross and net values are displayed. The translated value is displayed in lowercase by a lowercase angular pipeline in the template. It is therefore already taken into account.